### PR TITLE
Add a full readme explaining what this app is

### PR DESCRIPTION
**Pull request by idavidmcdonald**

### CODE REVIEW

**Comment by idavidmcdonald** [comment]
@rjbaker could I get you to review (when you have time) to make sure that this is factually correct. And then after that I might ask for a review from another dev on the team to make sure it's understandable/correct for someone who is not so familiar as this will likely be a rarely touched piece of infra/code.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+## What this app does
+
+An HTTP app that:
+
+- when given a certificate signing request, will sign it and return an issued certificate
+- provides certificate revocation lists (CRL) for our private certificate authorities
+
+## Why do we have this app
+
+We need to let the Mobile Network Operators (MNOs) get issued certificates signed by us for both TLS and VPN connections. We want to automate this process as much as possible so we avoid manual work every couple of months when certificates need rotating. This is especially important as their are multiple mobile networks to do this for. By having an HTTP endpoint for this the MNOs can request and get new certificates themselves.
```
Comment (rjbaker) [inline README.md:12]: _there_ are

**Review by rjbaker — APPROVED** [review]
Thanks for this 👍

File: README.md (hunk 1 above)
Comment (idavidmcdonald) [reply README.md]: Thanks, have added a new commit to fix this typo

**Pull request merged**